### DISCUSSION TITLE
preparing metadata(setup.py) ... error

### DISCUSSION THREAD

**Discussion by Zanejins** [nasa/fprime#1942] [Q&A]
when I install, that is pip install -r requirements.txt. The error occur.
![image](https://user-images.githubusercontent.com/98217021/227238191-89235d2e-90ae-4e34-8f78-aca468dbb118.png)

How to solve it. Thank you. My python version is 3.10

**Comment by LeStarch** [comment]
It looks like the machine you run on cannot access the above URL (https://github.com/fprime-community/fpp/releases/download/v1.0.2).

If you can access: https://github.com/fprime-community/fpp/releases//v1.0.2 then I'd try again as it may be a network issue.

**Reply by Zanejins** [reply]
Thank you for your reply. I can access https://github.com/fprime-community/fpp/releases//v1.0.2 , but still have this error.
![image](https://user-images.githubusercontent.com/98217021/229069424-f22d5afe-ee4f-4a5c-aca1-2b49eef66eb7.png)

**Reply by ThibFrgsGmz** [reply]
Can you access/download the package from the command line (e.g. wget)? If you are in a company, depending on their cybersecurity policy, the IT department may only give you access to GitHub through a secured web browser.

**Reply by ConstStar** [reply]
How can we solve this problem if we cannot download through wget?

**Comment by ConstStar** [comment]
I've had this situation happen, and I've tried many times but it hasn't been resolved

**Comment by LeStarch** [comment]
This issue should be fixed with the latest version of FPP as we restrctured our packages!

**Reply by ConstStar** [reply]
Thanks!
